### DISCUSSION TITLE
gmail.com problem related to PTR record

### DISCUSSION THREAD

**Discussion by mascor** [postalserver/postal#2916] [Help with using Postal] [answered]
Hi, I am observing strange behavior.
When I send an email with a limited number of recipients (100/200) the system works perfectly and emails to gmail.com are sent without any problem.
When I make sends with a higher number of emails 10,000/ 20,000 after several sends to gmail.com addresses, a "SOFT FAIL" error is shown and there are messages similar to this in the logs:
_"[<hidden_IPv6>] The IP address sending this message does not <hidden> have a PTR record, or the corresponding forward DNS entry does not <hidden> point to the sending IP. To protect our users from spam, mail has <hidden> been"_
I checked the PTR record of my email server with different tools and always got positive results. If the PTR record were not configured, the postal server would not be able to send emails to gmail.com but for mailings with a limited number of recipients it works.

Could the cause be a rate-limiting system that intervenes on the gmail.com side and limits email sending to a certain number of emails per unit of time or is it a problem with my IPv6 PTR record?

**Comment by willpower232** [comment, marked as answer]
If your server only has one IPv6 record and the PTR for it has been set correctly (you might have to check with the provider of the IP address as I think it is theirs to set) then yeah I guess it is possible this is a bit of a lie from gmail, they are being quite strict this year.
